<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>26</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="beginButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/player/icons/player/PlayerStart.png</normaloff>:/icons/player/icons/player/PlayerStart.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="previousButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/player/icons/player/PlayerRew.png</normaloff>:/icons/player/icons/player/PlayerRew.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="playButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/player/icons/player/PlayerPlay.png</normaloff>:/icons/player/icons/player/PlayerPlay.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/player/icons/player/PlayerFwd.png</normaloff>:/icons/player/icons/player/PlayerFwd.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="endButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/player/icons/player/PlayerEnd.png</normaloff>:/icons/player/icons/player/PlayerEnd.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Frame X of Y</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
